### DISCUSSION TITLE
fontview -> multiline (or wrap long line) for sampled text

### DISCUSSION THREAD

**Discussion by shula** [j2969719/doublecmd-plugins#38] [Ideas]
the sample text is on a single line.
a waste of space, if you want to see as many characters as possible in a glance.

Currently it is possible to show multi-line text, by copy-paste from a text editor
(which is how i did mine)
![image](https://github.com/user-attachments/assets/15de6ad7-8939-4b31-b7f1-347292074f00)

but it could be nice to have it built in (e.g. with "\n", or a hint to the user)

thanks!!

**Comment by j2969719** [comment]
tried changing qlabel to qtextedit, but it looks meh

you can set your custom string in the config (press * above the file panel >special dirs>go to dc special path>$DC_CONFIG_PATH>j2969719.ini) for line breaks use the html tag `<br>`

**Reply by shula** [reply]
Yes, indeed the INI file shows the \n markers. 
 <br> isn't interpreted in any special way.
the "\n" is affecting only from the INI file, not in the UI (the input textbox on top).
(which solves my problem, but still requires extra steps: open the ini, close DC, edit, save, start DC)
for me it's fine.
thanks!

![image](https://github.com/user-attachments/assets/da5b0b04-527e-4fe5-a191-9aa6b9aa8b06)

```
[fontview_qt_qt6.wlx]
bold=true
fontsize=51
italic=false
text=@String(\0שדגרק גהםצףץ\n 🐈🐱😄 531\n𝄞👪$✍#@\nABJQ ZNWIXE<br>\n the lazy dog\n)
```

**Reply by j2969719** [reply]
since you had line breaks copied from the editor - qt escaped them and the text became like `@String(...\n..)` so the html tags stopped working. try clearing the line and re-entering the text with html tags. in the master branch added a tooltip for QLineEdit, can also add a button for inserting `<br>`, but this will only eat up horizontal space, especially in `ctrl+q`
